### PR TITLE
Toyota: add FW engine for Brazilian Corolla Hybrid

### PR DESCRIPTION
**Pull request by AlexandreSato**
Test route `   4f50e44908cc46b8|2023-01-12--21-08-57  `

user: Eduardo Garufi#1576 (Discord)

### CODE REVIEW

**Comment by sshane** [comment]
Does this car have an electronic parking brake (it's able to stay stopped)?

**Comment by AlexandreSato** [comment]
Don't have electronic park brake !

Att
Alexandre Nobuharu Sato

Enviado do meu iPhone

Em 13 de jan. de 2023, à(s) 16:53, Shane Smiskol ***@***.***> escreveu:

﻿

Does this car have an electronic parking brake (it's able to stay stopped)?

**Comment by sshane** [comment]
Same as https://github.com/commaai/openpilot/pull/26693, I'll await for a stock route. Do you know if OP allows staying engaged at a stop?

**Comment by AlexandreSato** [comment]
route in DashCam mode: `          4f50e44908cc46b8|2023-01-12--23-33-58--0       `

OP don't allow engage at stops

Att
Alexandre Nobuharu Sato

Enviado do meu iPhone

Em 13 de jan. de 2023, à(s) 21:11, Shane Smiskol ***@***.***> escreveu:

﻿

Same as #26693<https://github.com/commaai/openpilot/pull/26693>, I'll await for a stock route. Do you know if OP allows staying engaged at a stop?

**Comment by AlexandreSato** [comment]
@sshane 

as requested here are the test routes:

Toyota stock long: `4f50e44908cc46b8|2023-01-14--11-58-35`

Openpilot long: `4f50e44908cc46b8|2023-01-14--12-05-32`

File: docs/CARS.md [hunk 1]
```diff
@@ -179,7 +179,7 @@ A supported vehicle is one that just works when you install a comma three. All s
 |Toyota|Corolla Cross (Non-US only) 2020-23|All|openpilot|17 mph|0 mph|[![star](assets/icon-star-full.svg)](##)|[![star](assets/icon-star-full.svg)](##)|Toyota||
 |Toyota|Corolla Cross Hybrid (Non-US only) 2020-22|All|openpilot|17 mph|0 mph|[![star](assets/icon-star-full.svg)](##)|[![star](assets/icon-star-full.svg)](##)|Toyota||
 |Toyota|Corolla Hatchback 2019-22|All|openpilot|0 mph|0 mph|[![star](assets/icon-star-full.svg)](##)|[![star](assets/icon-star-full.svg)](##)|Toyota|<a href="https://www.youtube.com/watch?v=_66pXk0CBYA" target="_blank"><img height="18px" src="assets/icon-youtube.svg"></img></a>|
-|Toyota|Corolla Hybrid 2020-22|All|openpilot|0 mph|0 mph|[![star](assets/icon-star-full.svg)](##)|[![star](assets/icon-star-full.svg)](##)|Toyota||
+|Toyota|Corolla Hybrid 2020-23|All|openpilot|0 mph|0 mph|[![star](assets/icon-star-full.svg)](##)|[![star](assets/icon-star-full.svg)](##)|Toyota||
```
Comment (sshane) [inline docs/CARS.md:182]: can you add a new entry for south america? it looks like the 2023's from Japan have Toyota security

Comment (AlexandreSato) [reply docs/CARS.md]: Done!

**Pull request merged**